food-truck-api/src/main/java/food/truck/api/search/IndexingService.java

```diff
@@ -0,0 +1,24 @@
+package food.truck.api.search;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j;
+
+import org.springframework.stereotype.Service;
+import org.hibernate.search.jpa.FullTextEntityManager;
+import org.hibernate.search.jpa.Search;
+import javax.persistence.EntityManager;
+import javax.transaction.Transactional;
+
+@Service
+@Log4j
+@RequiredArgsConstructor
+public class IndexingService {
+
+    private final EntityManager em;
```

Comment:
Is it autowiring this under the hood? I'm not seeing where it's getting instantiated, since it's being created by an autowired annotation.

Reply:
My guess would be that the autowired `IndexingService` gets passed the `EntityManager` when being created with the `@RequiredArgsConstructor`